process/block/preprocess/transactions.go

```diff
@@ -784,3 +884,79 @@ func (txs *transactions) IsInterfaceNil() bool {
 	}
 	return false
 }
+
+func (txs *transactions) computeGasConsumed(
+	senderShardId uint32,
+	receiverShardId uint32,
+	tx *transaction.Transaction,
+	txHash []byte,
+	gasConsumedByMiniBlockInSenderShard *uint64,
```

Comment:
maybe return these 2. and add to the local variable.

Reply:
These variables are modified inside this method and than used in the caller side